tests/test_local_storage.py

```diff
@@ -267,3 +267,20 @@ def test_local_storage_get_download_url_returns_none_on_empty_base(self) -> None
 
     def test_local_storage_rejects_multiple_query_values_for_download_url_key_setting(self) -> None:
         self.assert_rejects_multiple_query_values("/foo/bar/object.mp4", "download_url_base")
+
+    def test_local_storage_get_sanitized_uri(self) -> None:
```

Comment:
The test name should indicate what the expected behavior is. In this case, there's nothing "secret" in a local `file://` URI, so there is nothing to remove.